userland/examples/radio_rx/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <led.h>
+#include <radio.h>
+#include <timer.h>
+
+#define BUF_SIZE 60
+char packet_rx[BUF_SIZE];
+char packet_tx[BUF_SIZE];
+bool toggle = true;
+
+static void callback(__attribute__ ((unused)) int unused0,
```

Comment:
Sorry for bikeshedding, but I've noticed a lot of `__attribute__ ((unused))` in this codebase. In case you weren't aware, here's a more common way of dealing with unused parameters:

```c
static void callback(int unused0,
                     int unused1,
                     int unused2,
                     void* ud) {
    (void)unused0;
    (void)unused1;
    (void)unused2;
    (void)ud;
    led_toggle(0);
    radio_receive_callback(callback, packet_rx, BUF_SIZE);
}
```